Core/Tasks/Accelerometer.h

```diff
@@ -0,0 +1,11 @@
+//
+// Created by Taha Rashid on 2024-11-19.
+//
+
+#ifndef UOSM_TELEMETRY_PICO_ACCELEROMETER_H
+#define UOSM_TELEMETRY_PICO_ACCELEROMETER_H
+
+//to-do: need to create this function
+void InitAccelerometer();
```

Comment:
make sure to following the naming convention of the project.
This function should be called `accelerometer_init`

Reply:
Also get rid of the todo comment